CONTRIBUTING.md

```diff
@@ -247,33 +247,33 @@ Each PR should have one commit once it lands on `master`; this can be accomplish
 
 #### Major Release
 
-This major release process assumes that this release was preceded by release candidates. 
-If there were no release candidates, and you'd like to cut a major release directly from master, see below. 
+This major release process assumes that this release was preceded by release candidates.
+If there were no release candidates, and you'd like to cut a major release directly from master, see below.
 
 1. Start on the latest RC branch (`RCx/vX.X.0`).
 2. Run integration tests.
 3. Branch off of the RC branch (`git checkout -b release-prep`) and prepare the release:
-   - "Squash" changes from the changelog entries for the RCs into a single entry, 
```

Comment:
I believe @tessr left these space in for formatting reasons.

Reply:
The leading spaces are important for formatting (and sometimes the trailing spaces are too, honestly - markdown can be a pain). But in this case, I think it's fine to trim these trailing spaces.